README.md

```diff
@@ -421,14 +421,11 @@ Copyright (c) 2018-2023 tigattack, contributors and original authors.
 The CI will fail unless your code is [PEP8](https://www.python.org/dev/peps/pep-0008/) compliant.
 
 ```bash
-pip install -r requirements-ci.txt
-isort . # This will fix the order of imports
-black . # This will auto-format and fix a lot of common mistakes
-pylint * # This will show general pep8-violations
+pip install -r requirements-dev.txt -r requirements.txt
+pre-commit install
+pre-commit run --all-file
 ```
 
-If you use [VSCode](https://code.visualstudio.com/) you can use the tasks integrated into the repo to locally run the same tasks as our CI
```

Comment:
Since linting/formatting tasks are in pre-commit now, it probably makes sense to remove `.vscode/tasks.json` and add a suggestion to use the [pre-commit-vscode](https://marketplace.visualstudio.com/items?itemName=MarkLarah.pre-commit-vscode) extension if people still want to use VSCode tasks.